dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/random/BeanRandomizer.java

```diff
@@ -48,6 +49,7 @@
     public BeanRandomizer()
     {
         rand = aNewEnhancedRandomBuilder()
+            .randomize( ValueTypeOptions.class, new ValueTypeOptionsRandomizer() )
```

Comment:
Is there a reason to not just use a lambda instead of creating a new class?

Reply:
Agree, waste having an extra class just for this. Removed the ValueTypeOptionsRandomizer.java and made it into an lambda.